packages/fxa-payments-server/src/lib/coupon.test.ts

```diff
@@ -76,48 +76,39 @@ describe('lib/coupon', () => {
   });
 
   describe('checkCouponRepeating', () => {
-    let interval: string;
-    let interval_count: number;
-    let durationInMonths: number;
```

Comment:
I'm not sure if the runner is using concurrency? If yes, is it possible that the use of these variables by multiple tests is the cause of Flake? In which case the data driven conversion would be a fix.